radicle-cli/examples/workflow/5-patching.md

```diff
@@ -0,0 +1,69 @@
+Back to being the project maintainer.
+
+Changes have been proposed by another person (or peer) via a radicle patch.  To follow changes by another, we must 'track' them.
+
+```
+$ rad track did:key:z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk --alias bob
+✓ Tracking policy updated for z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk (bob)
+! Warning: fetch after track is not yet supported
+$ rad fetch
+✓ Fetching rad:z42hL2jL4XNk6K8oHQaSWfMgCL7ji from z6Mkt67…v4N1tRk..
+✓ Fetched repository from 1 seed(s)
+```
+
+Additionally, git must know about the new peer.  We need to add a new 'git
```

Comment:
I find the phrasing "git must know about the new peer" a bit strange, personally. I think about it more in terms of our working copy needing to know about the new peer.

Reply:
I'm uncertain about it.  This is configuration for the tooling.  The tooling needs to know where to fetch information from.

Reply:
Ya, I see your point :) Perhaps, "git must know about the new peer in the working copy of the project"?

Reply:
> Still a bit uncertain about it because its long

I wouldn't be worried about examples being long. Tutorial-like examples should definitely err on the side of being verbose, IMO.

> The reference about the 'working copy' comes out of left field for new users, we're familiar with the term because we engage with it all the time.

That's true, but it should definitely be a common name that is described. One of the conceptual tripping points will be the relation between the radicle storage and the working copy. I think a lot of people will be familiar with it in terms of remote git repositories and cloning a copy -- the difference being that the remote is actually stored locally :)

That all said, was mostly a suggestion and not a blocker :+1:

Reply:
I'm more concerned about anthropomorphizing git :)

I'd rather something like `Additionally, the new remote should be configured in git`

Reply:
"Additionally, we need to add a new 'git remote' to our working copy for the
peer.  Upcoming versions of radicle will not require this step."

Reply:
> I'm more concerned about anthropomorphizing git :)

Ah! You put your finger on what was setting my senses off! 😄